main.tf

```diff
@@ -29,6 +29,8 @@ module "sns_topic" {
   subscribers     = var.subscribers
   sqs_dlq_enabled = false
 
+  allowed_aws_services_for_sns_published = ["cloudwatch.amazonaws.com"]
```

Comment:
Btw, this is actually the default for this module https://github.com/cloudposse/terraform-aws-sns-topic/blob/master/variables.tf#L19 — not sure if you wanted to keep this in just so it's more declarative

Reply:
Yep, I just wanted to be explicit/clear when looking at this code :)

Reply:
This module could be also bumped to latest (`0.16.0`)

Reply:
Ah...good call I'll do that now.

Reply:
4bda0e04eeb3a58bfbe7502bef15477327afbead